EIPS/eip-1.md

```diff
@@ -265,11 +265,15 @@ The `description` field in the preamble:
 
 When referring to an EIP by number, it should be written in the hyphenated form `EIP-X` where `X` is the EIP's assigned number.
 
+### IETF RFCs
+
+When referring to an IETF RFC by number, it must be written in the non-hyphenated form `RFC X`, where `X` is the RFC's assigned number. Each RFC that is referenced in an EIP **MUST** be accompanied by a relative markdown link to the RFC (in the `assets/eip-1/RFCs` directory) the first time it is referenced, and **MAY** be accompanied by a link on subsequent references. If the RFC is not currently in that directory, a separate pull request must be submitted which only includes the RFC in that directory.
```

Comment:
Also, we want to store all RFCs in EIP-1's asset directory? That will reduce duplicates for sure, but it feels weird.

I'm not opposed to this pattern, just wanted to bring it up.

Reply:
Would it be better to have a separate `RFCs` directory in the assets folder? I strongly would like there to be a single place that contains all the RFCs.

Reply:
So just `assets/RFCs`?   Fetching them is yet another thing a bot could do ;)